examples/simple/index.ts

```diff
@@ -6,88 +6,13 @@ import * as google from "@pulumi/google-native";
 const project = "pulumi-development";
 const region = "us-central1";
 
-const runName = "run-native";
```

Comment:
To be clear, removing this because this is covered by the cloudrun example?

Reply:
Yes, I removed duplication